### ISSUE TITLE
Generate Estimate: Can this autopopulate rather than click the button

### ISSUE THREAD

**Issue by dpdavids** [icefoganalytics/travel-authorization#253]
Relates to:
https://travel-auth-dev.ynet.gov.yk.ca/my-travel-requests/250/estimate/edit

# Context

**Is your feature request related to a problem? Please describe.**
When I move to step 3. I need to select the Generate estimate button.  This is not completely intuitive and it is a step that must happen. I am wondering when I move to step 3 can it just generate the estimates for me?

**Describe the solution you'd like**
on continue button press  on step 2. can the estimates appear on step 3.  if it takes a bit of time can we just put in a loading circle with the note from the generate pop-up.  

**Describe alternatives you've considered**

**Additional context**

**Comment by klondikemarlen** [comment]
Yeah, for sure, though it might require some re-egineering. That said, ability to trigger extra actions on a per-step basis, will add a lot of value as it would smooth out a bunch of other UI problems.
